vendor/src/golang.org/x/net/proxy/direct.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2011 The Go Authors. All rights reserved.
```

Comment:
This package was already vendored but the `proxy` module had been pruned, so I've added it in.